### PR TITLE
Refactor(eos_designs)!: Change MLAG VLAN names and SVI descriptions

### PR DESCRIPTION
**Pull request by ClausHolbechArista**
## Change Summary

Change MLAG VLAN names and SVI descriptions

## Component(s) name

`arista.avd.eos_designs`

## Proposed changes

- Adding new default descriptions for MLAG SVIs and VLANs
- Add templating support for all
- Add keys with default templates:
  - `mlag_peer_svi_description`
  - `mlag_peer_l3_svi_description`
  - `mlag_peer_vlan_name`
  - `mlag_peer_l3_vlan_name`
- Updated porting guide

## How to test

- Update molecule scenarios and examples
- Updated custom description logic in unit tests since we now expose the mlag svi descriptions

### User Checklist

- N/A

### Repository Checklist

- [x] My code has been rebased from devel before I start
- [x] I have read the [**CONTRIBUTING**](https://avd.sh/en/latest/docs/contribution/overview.html) document.
- [x] My change requires a change to the documentation and documentation have been updated accordingly.
- [x] I have updated [molecule CI](https://github.com/aristanetworks/avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly. (check the box if not applicable)

### CODE REVIEW

File: ansible_collections/arista/avd/docs/porting-guides/5.x.x.md [hunk 1]
```diff
@@ -502,6 +502,56 @@ The description can be reverted manually if needed:
...
++mlag_peer_l3_svi_description: "MLAG_PEER_L3_PEERING"
+```
+
+The description setting supports a template. See the input variables documentation for available fields.
+
+### Default VLAN names are changed for more consistency
+
+#### MLAG peer VLAN
+
+The name of the MLAG peer VLAN (VLAN 4094 by default)  changes to `MLAG`. Previously the name was `MLAG_PEER`.
```
Comment (gmuloc) [inline ansible_collections/arista/avd/docs/porting-guides/5.x.x.md]:
```suggestion
The name of the MLAG peer VLAN (VLAN 4094 by default) changes to `MLAG`. Previously the name was `MLAG_PEER`.
```

File: ansible_collections/arista/avd/docs/porting-guides/5.x.x.md [hunk 2]
```diff
@@ -502,6 +502,56 @@ The description can be reverted manually if needed:
...
+
+The description can be reverted manually if needed:
```
Comment (gmuloc) [inline ansible_collections/arista/avd/docs/porting-guides/5.x.x.md]:
```suggestion
The name can be reverted manually if needed:
```

File: ansible_collections/arista/avd/docs/porting-guides/5.x.x.md [hunk 3]
```diff
@@ -502,6 +502,56 @@ The description can be reverted manually if needed:
...
++mlag_peer_vlan_name: "MLAG_PEER"
+```
+
+The setting supports a template. See the input variables documentation for available fields.
+
+#### MLAG L3 peering VLAN
+
+The name of the MLAG L3 peering VLAN (VLAN 4093 by default) changes to `MLAG_L3`. Previously the name was `LEAF_PEER_L3`.
+
+The description can be reverted manually if needed:
```
Comment (gmuloc) [inline ansible_collections/arista/avd/docs/porting-guides/5.x.x.md]:
```suggestion
The name can be reverted manually if needed:
```

**Review by gmuloc — APPROVED** [review]

**Review by carlbuchmann — APPROVED** [review]
LGTM - tested porting guide with ATD lab repo.

**Pull request merged**